docs/getting-started-guide.md

```diff
@@ -7,7 +7,7 @@
 $ make
 go build -o bin/rktlet ./cmd/server/main.go
 
-$ sudo ./bin/rktlet -v=4 -alsologtostderr
+$ sudo ./bin/rktlet -v=4 --alsologtostderr
```

Comment:
@euank maybe by default we should set `logtostderr=true` so we can almost avoid this additional flag.